draft-ietf-oauth-transaction-tokens.md

```diff
@@ -497,6 +500,14 @@ A requester MAY use a self-signed JWT as a `subject_token` value. In that case,
 
 The self-signed JWT MAY contain other claims.
 
+### Unsigned JSON Object Subject Token Type {#unsigned-json-subject-token-type}
+A requester MAY use an unsigned JSON object as a `subject_token` value. In that case, the requester MUST set the `subject_token_type` value to: `urn:ietf:params:oauth:token-type:unsigned_json`. The JSON object in the subject token MUST contain the following fields:
```

Comment:
How exactly is the  unsigned JSON encoded? Is it just a JSON string or is it at least base64urll encoded? We may need  to  mention this.

Reply:
That's good callout to encode base64. Our language can be similar to what it says here in https://www.ietf.org/archive/id/draft-jones-json-web-token-02.html#rfc.section.7

Reply:
I believe a JWT is always Base64url encoded. It is either a JWS or a JWE, and those specs specify that it is encoded, e.g. in this section: https://datatracker.ietf.org/doc/html/rfc7515#section-5

Reply:
Yes JWTs are always base64 encoded, but we do not use an unsigned JWT here, but an unsigned JSON object. This is different.